### ISSUE TITLE
Localization doesn't work and throws errors

### ISSUE THREAD

**Issue by cremor** [orionshock/sick-of-clicking-dailies#1]
**Describe the bug**
Seems like the packager doesn't work correctly. The files in the "localization" folder and the titles in the toc file are not populated with localization entires.

**To Reproduce**
1. Check the files in the localization directory of the released package. Notice the missing data.
2. Check the addon titles in the toc file of the released package. Notice the missing data.
3. Check the ingame addon list. Notice the wrong addon titles and descriptions.
4. Login
5. Notice the error messages and the missing localizations.

**Expected behavior**
Localizations should be there. Or at least fallback to english without throwing an error or showing weird titles.

**Screenshots**
![grafik](https://user-images.githubusercontent.com/116314/125188533-70963100-e234-11eb-8c44-3b2ef594cd68.png)

**Version and Client:**
 - Version: v9.1.0-retail
 - Client: Retail
 
**Additional context**
Example error that is thrown on login:

```
1x AceLocale-3.0-6: SickOfClickingDailies: Missing entry for 'Auto Complete Daily or Weekly?'
[string "@SickOfClickingDailies\options-Optionen.lua"]:260: in function `CreateInteractionFrame'
[string "@SickOfClickingDailies\options-Optionen.lua"]:23: in function <SickOfClickingDailies\options.lua:22>
[string "=[C]"]: ?
[string "@BagSync\libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:70: in function <...ceBagSync\libs\AceAddon-3.0\AceAddon-3.0.lua:65>
[string "@BagSync\libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:498: in function `InitializeAddon'
[string "@BagSync\libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:623: in function <...ceBagSync\libs\AceAddon-3.0\AceAddon-3.0.lua:615>
```

**Comment by cremor** [comment]
Maybe it isn't the packager's fault but the fact that the localization system is diabled? Because the readme links to https://www.wowace.com/projects/sick-of-clicking-dailies/localization which doesn't exist any more.

**Comment by cremor** [comment]
@orionshock Since you are the owner of the CurseForge project I assume you'd have to enable the localization system there again?

**Comment by cremor** [comment]
@orionshock This is partly fixed in v9.1.2-retail, which causes it to now be even more broken 😢 
The localization entries are back in the packaged file, but the main TOC file now contains the following in line 12: `]=]` This causes the whole loading of the addon to fail. Everything is fine if that line is removed.

I'm not sure where that line comes from, but a guess would be the zhTW description text for the TOC file, because that is the last line before the broke one and that localization phrase contains a line break at the end.

**Comment by cremor** [comment]
Fixed in v9.1.2a-retail, thanks!
